src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,13 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed === true,
```

Comment:
You don't need to do this check `completed === true`. Completed is already `true` or `false`